Makefile

```diff
@@ -0,0 +1,29 @@
+all: build check
+
+check: fmt vet lint ## Check your code
+
+revendor:
+	rm glide.lock
+	glide install --strip-vendor
+	glide-vc --use-lock-file --no-tests --only-code
+
+terraform:
+	hack/get-terraform.sh
+
+build: ## build installer binary
+	hack/build.sh
+
+#test: # Run unit test
+#	go test -race -cover ./cmd/...
```

Comment:
This should be [over `./pkg/...`][1].  Otherwise this looks good; why comment it out?

[1]: https://github.com/openshift/release/blob/e48cca67d52e4a4a645728f0759fc24bf4e7e0bb/ci-operator/config/openshift/installer/master.yaml#L45